runtime/compiler/p/codegen/J9TreeEvaluator.cpp

```diff
@@ -2117,7 +2117,17 @@ TR::Register *J9::Power::TreeEvaluator::asynccheckEvaluator(TR::Node *node, TR::
 
 TR::Register *J9::Power::TreeEvaluator::instanceofEvaluator(TR::Node *node, TR::CodeGenerator *cg)
    {
-   return VMinstanceOfEvaluator(node, cg);
+   TR::Compilation * comp = cg->comp();
```

Comment:
The PR does not explain why this code was added. In the current code, instanceofEvaluator calls VMinstanceOfEvaluator(node, cg) which in turn calls VMinstanceOfEvaluator2(node, cg) (unless some old env var is set). With this commit, it will not always call VMinstanceOfEvaluator2(node, cg). Was it intentional?

Reply:
@gita-omr I think this is a mistake that happened when I rebased this branch. Since this PR is pretty old, there were several conflicts, and something weird must have happened when I tried to resolve them. It should be fixed now

Reply:
Great! Makes more sense now :)